writers/README.md

```diff
@@ -0,0 +1,16 @@
+# Writers package
+
+Writers package provides an implementation of various `message consumers`.
+Message consumers are service that consume normalized (in `SenML` format)
+Mainflux messages and store them in specific data store.
+
+For an in-depth explanation of the usage of `writers`, as well as thorough
+understanding of Mainflux, please check out the [official documentation][doc].
+
+## Configuration
```

Comment:
All of these sections are writer-specific, and they should be removed from this file.